tests/foreman/cli/test_satellitesync.py

```diff
@@ -306,48 +327,424 @@ def set_importing_org(self, product, repo, cv):
+            'name': cls.exporting_repo_name,
+            'download-policy': 'immediate',
+            'product-id': product['id']
+        })
+        Repository.synchronize({'id': cls.exporting_repo['id']})
+        cls.exporting_cv_name = gen_string('alpha')
+        cls.exporting_cv, cls.exporting_cvv_id = ContentViewSync._create_cv(
+            cls.exporting_cv_name, cls.exporting_repo, cls.exporting_org)
+
+    def tearDown(self):
+        """Deletes Directory created for CV export Test during setUp"""
+        super(ContentViewSync, self).tearDown()
+        ssh.command('rm -rf {}/*'.format(self.export_base))
+
     @run_only_on('sat')
     @tier3
-    @upgrade
     def test_positive_export_import_cv(self):
         """Export CV version contents in directory and Import them.
 
-        :id: b08e9f24-f18e-43b7-9189-ad7b596ccb5b
+        :id: b4fb9386-9b6a-4fc5-a8bf-96d7c80af93e
 
         :steps:
 
-            1. Export whole CV version contents to a directory
-            3. Import those contents from some other org/satellite.
+            1. Create product and repository with custom contents.
+            2. Sync the repository.
+            3. Create CV with above product and publish.
+            4. Export CV version contents to a directory
+            5. Import those contents from some other org/satellite.
 
         :expectedresults:
 
-            1. Whole CV version contents has been exported to directory
-            2. All The exported contents has been imported in org/satellite.
+            1. CV version custom contents has been exported to directory
+            2. All The exported custom contents has been imported in org/satellite
 
         :CaseLevel: System
         """
         ContentView.version_export({
-            'export-dir': '{}'.format(self.export_dir),
+            'export-dir': '{}'.format(self.export_base),
             'id': self.exporting_cvv_id
         })
-        exported_tar = '{0}/export-{1}.tar'.format(self.export_dir, self.exporting_cvv_id)
+        exported_tar = '{0}/export-{1}.tar'.format(self.export_base, self.exporting_cvv_id)
         result = ssh.command("[ -f {0} ]".format(exported_tar))
         self.assertEqual(result.return_code, 0)
         exported_packages = Package.list({'content-view-version-id': self.exporting_cvv_id})
-        self.set_importing_org(self.exporting_prod, self.exporting_repo, self.exporting_cv)
+        self.set_importing_org(
+            self.exporting_prod_name, self.exporting_repo_name, self.exporting_cv_name)
         ContentView.version_import({
             'export-tar': exported_tar,
             'organization-id': self.importing_org['id']
         })
-        importing_cvv_id = ContentView.info({
+        importing_cvv = ContentView.info({
             u'id': self.importing_cv['id']
+        })['versions']
+        self.assertTrue(len(importing_cvv) >= 1)
+        imported_packages = Package.list({'content-view-version-id': importing_cvv[0]['id']})
```

Comment:
can we assert that len(imported_packages) or len(exported_packages) > 0 to be sure there is some packages there